packages/components/pages/f-contact-preferences/src/components/_tests/ContactPreferences.test.js

```diff
@@ -143,6 +166,34 @@ describe('ContactPreferences Component', () => {
             // Assert
             expect(wrapper.vm.isFormDirty).toEqual(false);
         });
+
+        it('should not call initialise method if the authorisation has not completed', () => {
+            // Arrange
+            sutProps = { ...sutProps, isAuthFinished: false };
+
+            // Act
+            wrapper = mountContactPreferences();
+
+            // Assert
+            expect(storeActions.loadPreferences).not.toHaveBeenCalled();
```

Comment:
The test's name refers to the initialise method, but the assert refers to something else. It's a bit confusing 😅

Reply:
It was simpler to just check if a method inside the `Initialise()` had been called but I can set up a new spy 👍